Emby.XmlTv/Emby.XmlTv/Classes/XmlTvReader.cs

```diff
@@ -495,16 +495,33 @@ public void ProcessEpisodeNum(XmlReader reader, XmlTvProgram result)
                     ParseMovieDbSystem(reader, result);
                     break;
                 case "SxxExx":
-                    // TODO
-                    // <episode-num system="SxxExx">S03E12</episode-num>
-                    reader.Skip();
+                    ParseSxxExxSystem(reader, result);
                     break;
                 default: // Handles empty string and nulls
                     reader.Skip();
                     break;
             }
         }
+        public void ParseSxxExxSystem(XmlReader reader, XmlTvProgram result)
+        {
+            //// <episode-num system="SxxExx">S012E32</episode-num
+            string value = reader.ReadElementContentAsString();
+            Regex exp = new Regex("s([0-9]+)e([0-9]+)",RegexOptions.IgnoreCase);
+            Match res = new Regex("s([0-9]+)e([0-9]+)", RegexOptions.IgnoreCase).Match(value);
```

Comment:
Why did you change to a `Regex` instance instead of the static one?
Depending on the amount of times this function is executed this can have big performance consequences.
(Also, you are creating 2 of them :p)

Reply:
Sorry was going through the requested changes quickly yesterday.
https://github.com/jellyfin/jellyfin/pull/1009/commits/49f4be6e7949a9cd269980aa97792e7453baf6c6

Should address the issues.